### PR TITLE
Create test_on_pull2.md

### PR DESCRIPTION
**Pull request by santrancisco**

### CODE REVIEW

File: .github/workflows/attack2.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+permissions: write-all
+
+on:
+  pull_request:
+    types: [locked]
+  pull_request_review:
+    types: [edited, dismissed, submitted]
+  issue_comment:
```
Comment (robot2051) [inline .github/workflows/attack2.yml:8]: wtf

File: .github/workflows/wtf.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+on:
+  pull_request:
+    types: [locked]
+  pull_request_review:
+    types: [created, edited, dismissed, submitted]
+  issue_comment:
+    types: [created, edited, dismissed, submitted]
+  pull_request_review_comment:
+    types: [created, edited, dismissed, submitted]
```
Comment (robot2051) [inline .github/workflows/wtf.yml:11]: trigger this

Comment (santrancisco) [reply .github/workflows/wtf.yml:11]: test

Comment (robot2051) [reply .github/workflows/wtf.yml:11]: wtf

**Pull request closed without merging**